sdk/tests/conftest.py

```diff
@@ -73,11 +73,15 @@ def port():
 
 @pytest.fixture(autouse=True)
 def vcr_config():
+    def ignore_some_localhost(request: VcrRequest):
+        if (request.host == "0.0.0.0" or request.host=="localhost") and port != 11434:
```

Comment:
And this is the hack for local Ollama

Reply:
Is this backwards? I assume you mean to ignore all traffic on port 11434 and you want to ignore that.

Reply:
We want to ignore all trafic on localhost and only allow traffic on port 11434 since that is the port ollama runs on.